content/en/getting-started/configuration.md

```diff
@@ -394,6 +418,18 @@ See [Content Management](/content-management/urls/#permalinks).
 
 (`bool`) Whether to pluralize automatic list titles. Applicable to section pages. Default is `true`.
 
+###### printI18nWarnings
+
+(`bool`) Print missing translations. Default is `false`.
+
+###### printPathWarnings
+
+(`bool`) Print warnings on duplicate target paths etc. Default is `false`.
```

Comment:
```suggestion
(`bool`) Whether to print warnings to the console when Hugo publishes two or more files to the same path. Default is `false`.
```

Reply:
my suggestion - see global comment:
(`bool`)  Whether to log WARNINGs when Hugo publishes two or more files to the same path. Default is `false`.